spyder/app/mainwindow.py

```diff
@@ -1453,23 +1453,23 @@ def post_visible_setup(self):
         window.screenChanged.connect(self.handle_new_screen)
         self.screen = self.window().windowHandle().screen()
         self.screen.logicalDotsPerInchChanged.connect(
-            self.show_DPI_change_message)
+            self.show_dpi_change_message)
 
         # Notify that the setup of the mainwindow was finished
         self.sig_setup_finished.emit()
 
     def handle_new_screen(self, screen):
         """Connect DPI signals for new screen."""
         self.screen.logicalDotsPerInchChanged.disconnect(
-            self.show_DPI_change_message)
+            self.show_dpi_change_message)
         self.screen = screen
         self.screen.logicalDotsPerInchChanged.connect(
-            self.show_DPI_change_message)
+            self.show_pdi_change_message)

```

Comment:
You made a little mistake here: it needs to be `show_dpi_change_message`, not `show_pdi_change_message`.